account-sync/Dockerfile

```diff
@@ -0,0 +1,10 @@
+ARG PYTHON_VERSION=3.11
```

Comment:
I'm not attached to having this account-sync container built from this repository, but given the recent drop of Debian support for the `nss-pam-ldap` library in favour of `nss-pam-ldapd`, and as the configuration is so nestled deep in the inner workings of the container, this seemed a reasonable place for it to be built from to ensure it used a compatible version of the base container image.

Reply:
We could also just have a standalone repo in DiamondLightSource for this. It feels like the cadence of release of account-sync vs python-copier-template would be wildly different. Account sync could be built occasionally (almost never after the first time?) and if you update it then you need to update the version tag that copier template refers to.

Reply:
I think I prefer that, because otherwise there will be many releases of the account-sync image with the same contents.

Reply:
I'll probably end up moving it to its own repository and using some release automation that I want to try out to keep it up to date.